# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-register.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

 2:27.46 /usr/src/dirtier/src.arcticfox.arm64.250812.231127.7029/Arctic-Fox-46.1/js/src/jit/none/MacroAssembler-none.h:87:63: error: 'invalid_reg' is not a member of 'js::jit::Register'
 2:27.46    87 | static constexpr Register WasmTableCallIndexReg = { Register::invalid_reg };

Fixed in main: https://github.com/rmottola/Arctic-Fox/commit/151a3c0b980e97247854dc74e7b67e5537253c32

--- Arctic-Fox-46.1/js/src/jit/none/MacroAssembler-none.h.vanilla	2025-08-12 23:16:55.783906360 +0200
+++ Arctic-Fox-46.1/js/src/jit/none/MacroAssembler-none.h	2025-08-12 23:17:08.099372972 +0200
@@ -84,7 +84,7 @@
 
 static constexpr Register WasmTableCallScratchReg = { Registers::invalid_reg };
 static constexpr Register WasmTableCallSigReg = { Registers::invalid_reg };
-static constexpr Register WasmTableCallIndexReg = { Register::invalid_reg };
+static constexpr Register WasmTableCallIndexReg = { Registers::invalid_reg };
 static constexpr Register WasmTlsReg = { Registers::invalid_reg };
 
 static constexpr uint32_t ABIStackAlignment = 4;
